osg_configure/configure_modules/rsv.py

```diff
@@ -1086,70 +1082,6 @@ def split_2d_list(self, item_list):
         # We shouldn't reach here, but just in case...
         return array
 
-    def _set_gratia_collector(self, collector):
```

Comment:
Isn't this section for the gratia (ie, gracc) collector?  (As opposed to the gratia-consumer in rsv?)

Reply:
Nope. The configuration for gratia stuff lives in gratia.py.

Reply:
Oh wow, so all this junk is just for the metric probe.

Ok, good riddance then!